src/drivers/warmup.ts

```diff
@@ -0,0 +1,129 @@
+import { createAction } from '@reduxjs/toolkit'
+import axios from 'axios-observable'
+import { Epic } from 'redux-observable'
+import { of, race, throwError } from 'rxjs'
+import { catchError, delay, filter, map, mapTo, mergeMap, withLatestFrom } from 'rxjs/operators'
+
+/**
+ * This driver handles cache warmups for a HTTP endpoint, in this case, specifically for v1 external adapters.
+ *
+ * This driver gets events that are sent in from v1 adapters, then tracks when each EA cache would get cold via configured TTL.
+ * If the cache is ever determined to turn cold, a warmup response is sent to that particular EA to keep its cache warm, avoiding
+ * performance penalties of a cache bust.
+ */
+
+interface WarmupRequestRecievedPayload {
```

Comment:
Valuable info to get from EA could be "was this a cache HIT or MISS?". We could use that info to implement a smarter warmup driver strategy.

A simple strategy is just to hit all warmup registered EAs, in short and regular intervals. Additional info like this could enable us to improve on that strategy.